packages/react/src/components/MultiSelect/__tests__/__snapshots__/FilterableMultiSelect-test.js.snap

```diff
@@ -114,7 +114,6 @@ exports[`MultiSelect.Filterable should render 1`] = `
                 aria-owns={null}
                 className="bx--list-box__field"
                 data-toggle={true}
-                disabled={false}
```

Comment:
What does removing the `disabled={false}` line do? Just curious, trying to learn more about what's going on here 🙂

Reply:
this is just a snapshot update, so when this component is rendered in this test it no longer has the `disabled` prop

Reply:
Okay cool, good to know. Thank you!